workers/sentencereviewer/testhelpers/postgres_testcontainer.go

```
@@ -0,0 +1,131 @@
+			language TEXT NOT NULL CHECK (language in ('English', 'Turkish')),
+			repetitions SMALLINT NOT NULL DEFAULT 0,
+			ease_factor REAL NOT NULL DEFAULT 2.5 CHECK (ease_factor >= 1.29),
+			"interval" SMALLINT NOT NULL DEFAULT 0 CHECK (interval >= 0),
+			next_review_instant TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
+			UNIQUE (sentence_id, user_id)
+		);
+	`)
+
+	if err != nil {
+		dbConn.Close()
+		container.Terminate(ctx)
```

golangci-lint (1.64.8)

Error return value of `container.Terminate` is not checked

(errcheck)